third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -3107,6 +3109,7 @@ RecordReplayRegisterV8Inspector(v8_inspector::V8Inspector* inspector,
     // For now we only connect to the first frame.
     static int ContextGroupId = 1;
 
+    recordreplay::AutoDisallowEvents disallow("RecordReplayRegisterV8Inspector");
```

Comment:
Can we add IsReplaying() to the test above this?

Reply:
I guess you mentioned in the past, that we did not want the debugger active during recording?

I'll add it and run a few tests. Fingers crossed, it won't cause a lot of new divergences.

Reply:
As expected, there is some more follow-up coming out of this: https://linear.app/replay/issue/RUN-1596/mismatch-in-v8consolemessagestorageaddmessage-due-to-replay-only